src/test/java/HomeWork22.java

```diff
@@ -0,0 +1,25 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork22 extends BaseTest{
+    @Test
+    public void renamePlaylist() throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.